### PR TITLE
Update sample_products.xml to include attachments

### LINKED ISSUE #21663: New sample data does not work with images or variable products
**Describe the bug**
Hi there,
I'm not sure what's happened but the `sample-data/sample_products.xml` test data (previously `dummy-data/dummy-products.xml`) no longer works for images and variable products, as of https://github.com/woocommerce/woocommerce/commit/b3af5c808d7df552b88b6989fc20368b4795aff7

**To Reproduce**

Import https://raw.githubusercontent.com/woocommerce/woocommerce/master/sample-data/sample_products.xml on a clean woocommerce install

new data:
```
curl -L https://raw.githubusercontent.com/woocommerce/woocommerce/master/sample-data/sample_products.xml -o sample_data_new.xml
wp import sample_data_new.xml --authors=create
```

old data:
```
curl -L https://raw.githubusercontent.com/woocommerce/woocommerce/c81b3cf1655f9983db37bff750cb5baae3c3236e/dummy-data/dummy-products.xml -o sample_data_old.xml
wp import sample_data_old.xml --authors=create
```

**Screenshots**

Old test data:
Imgur

New test data
Imgur

**Expected behavior**

I would expect images and variations to work the same way they did with the test data previously.

As you can see some, but not all images work. (As far as I remember the images didn't work at all when I imported the new test data originally, they may only be there because the image filenames are the same).

Also you can see that the variable products didn't import correctly because it doesn't display any prices, and when you look at the product it doesn't have any variations (but it does correctly set the product type as variable)

**Isolating the problem (mark completed items with an [x]):**
- [x] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**WordPress Environment**
<details>
```
### WordPress Environment ###

Home URL: http://localhost:8083
Site URL: http://localhost:8083
WC Version: 3.4.6
Log Directory Writable: ✔
WP Version: 4.9.8
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: ✔
WP Cron: ✔
Language: en_US
External object cache: –

### Server Environment ###

Server Info: Apache
PHP Version: 7.1.22 - We recommend using PHP version 7.2 or above for greater performance and security. How to update your PHP version
PHP Post Max Size: 50 MB
PHP Time Limit: 30
PHP Max Input Vars: 1000
cURL Version: 7.52.1
OpenSSL/1.0.2l

SUHOSIN Installed: –
MySQL Version: 5.5.5-10.3.9-MariaDB-1:10.3.9+maria~bionic
Max Upload Size: 25 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 3.4.6
WC Database Prefix: wp_
MaxMind GeoIP Database: ❌ The MaxMind GeoIP Database does not exist - Geolocation will not function. You can download and install it manually from https://dev.maxmind.com/geoip/geoip2/geolite2/ to the path: . Scroll down to "Downloads" and download the "MaxMind DB binary
gzipped" file next to "GeoLite2 Country". Please remember to uncompress GeoLite2-Country_xxxxxxxx.tar.gz and upload the GeoLite2-Country.mmdb file only.

Total Database Size: 9.80MB
Database Data Size: 8.54MB
Database Index Size: 1.26MB
wp_woocommerce_sessions: Data: 5.52MB + Index: 0.13MB
wp_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB
wp_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB
wp_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.05MB
wp_woocommerce_order_items: Data: 0.02MB + Index: 0.02MB
wp_woocommerce_order_itemmeta: Data: 0.02MB + Index: 0.03MB
wp_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB
wp_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB
wp_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB
wp_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB
wp_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB
wp_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB
wp_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB
wp_woocommerce_log: Data: 0.02MB + Index: 0.02MB
wp_commentmeta: Data: 0.02MB + Index: 0.03MB
wp_comments: Data: 0.02MB + Index: 0.09MB
wp_links: Data: 0.02MB + Index: 0.02MB
wp_options: Data: 1.09MB + Index: 0.11MB
wp_postmeta: Data: 1.05MB + Index: 0.17MB
wp_posts: Data: 0.22MB + Index: 0.13MB
wp_termmeta: Data: 0.02MB + Index: 0.03MB
wp_terms: Data: 0.02MB + Index: 0.03MB
wp_term_relationships: Data: 0.02MB + Index: 0.02MB
wp_term_taxonomy: Data: 0.02MB + Index: 0.03MB
wp_usermeta: Data: 0.20MB + Index: 0.03MB
wp_users: Data: 0.02MB + Index: 0.05MB
wp_wc_download_log: Data: 0.02MB + Index: 0.03MB
wp_wc_webhooks: Data: 0.02MB + Index: 0.02MB

### Post Type Counts ###

attachment: 55
json_consumer: 2
page: 6
post: 306
product: 18
revision: 285

### Security ###

Secure connection (HTTPS): ❌
					Your store is not using HTTPS. Learn more about HTTPS and SSL Certificates.
Hide errors from visitors: ❌Error messages should not be shown to visitors.

### Active Plugins (5) ###

JSON Basic Authentication: by WordPress API Team – 0.1
WP REST API - OAuth 1.0a Server: by WP REST API Team – 0.3.0
WP REST API: by WP REST API Team – 2.0-beta15
WooCommerce: by Automattic – 3.4.6 – 3.5.0 is available
WordPress Importer: by wordpressdotorg – 0.6.4

### Settings ###

API Enabled: ✔
Force SSL: –
Currency: GBP (£)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)


### WC Pages ###

Shop base: #4 - /shop/
Cart: #5 - /cart/
Checkout: #6 - /checkout/
My account: #7 - /my-account/
Terms and conditions: ❌ Page not set

### Theme ###

Name: Twenty Seventeen
Version: 1.7
Author URL: https://wordpress.org/
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme
WooCommerce Support: ✔

### Templates ###

Overrides: –
```
</details>

### PR DESCRIPTION
**Pull request by kloon**
### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

The XML import file included with the sample data current does not include attachments which when imported results in no images. This PR include an updated xml import file with the all the required data to result in a successful product import.

Closes #21663

### How to test the changes in this Pull Request:

1. Start with a clean WP installations with WooCommerce plugin and no products or images imported previously.
2. Import products via the WP Importer, Tools -> Import
3. Ensure that images are imported and displaying.

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?

### Changelog entry

> Fix - XML Sample Data import file was missing images, not has all the required data.

### CODE REVIEW

**Comment by jobthomas** [comment]
I'm not sure how easy it is to get rid of that, but when I previously compiled the XML file, I got rid of all the personal details in the file (e.g. there are several email addresses of team members in there).

**Comment by jobthomas** [comment]
Thanks for that update @kloon

**Comment by claudiulodro** [comment]
screen shot 2018-10-29 at 12 11 00 pm
I didn't get images for almost all products on a clean install. The XML looks good (I think). I'll investigate further and see what I discover.

File: sample-data/sample_products.xml [hunk 1]
```diff
@@ -1,3164 +1,5770 @@
...
+  <wp:postmeta>
+    <wp:meta_key>_sku</wp:meta_key>
+    <wp:meta_value><![CDATA[woo-beanie]]></wp:meta_value>
+  </wp:postmeta>
+  <wp:postmeta>
+    <wp:meta_key>_regular_price</wp:meta_key>
+    <wp:meta_value><![CDATA[20]]></wp:meta_value>
+  </wp:postmeta>
+  <wp:postmeta>
+    <wp:meta_key>_sale_price</wp:meta_key>
```
Comment (claudiulodro) [inline sample-data/sample_products.xml]: There seems to be duplicate info for all the metas. For example if you look at line 89 and line 213 the `_sale_price` meta appears twice.

File: sample-data/sample_products.xml [hunk 2]
```diff
@@ -1,3164 +1,5770 @@
...
+    <wp:meta_value><![CDATA[3.1.1]]></wp:meta_value>
+  </wp:postmeta>
+  <wp:postmeta>
+    <wp:meta_key>_price</wp:meta_key>
+    <wp:meta_value><![CDATA[35]]></wp:meta_value>
+  </wp:postmeta>
+  <wp:postmeta>
+    <wp:meta_key>_thumbnail_id</wp:meta_key>
+    <wp:meta_value><![CDATA[190]]></wp:meta_value>
+  </wp:postme
```
Comment (claudiulodro) [inline sample-data/sample_products.xml]: Most of the attachments are missing. Look at the lines below this comment and you will see there are only a small handful of the attachments. This seems to be the source of the missing images on import, as the attachments in this file seem to import and associate with products fine.

**Review by claudiulodro — DISMISSED** [review]
Noticed a couple issues in the file.

**Comment by mikejolley** [comment]
Ping @kloon

**Comment by kloon** [comment]
OK, this is ready for retesting. Think this is the cleanest I can get the file. Tested on a clean install with no media and all images were downloaded and assigned to products correctly.

**Review by mikejolley — APPROVED** [review]

**Pull request merged**